h5p-bildetema/src/components/Bildetema/Bildetema.module.scss

```diff
@@ -8,6 +8,7 @@
 
   :focus {
     outline: 3px solid $orange;
+    z-index: 9999; // prevent adjacent elements from overlapping the outline
```

Comment:
A z-index above 10-20 is oftentimes a code smell because we might at some point override it. However, as this is a special case I think we can resort to having an enormous z-index. 

The `z-index` property only affects elements that have a `position` other than the default `static` (and a few other cases), which means that this won't do anything in most cases. Perhaps we should fix this where it is a problem instead? That means that some cases might go unnoticed, but we can't set `position` here.

In any case: We should make a SCSS variable for this and set the value in a common variables file somewhere. It sometimes becomes crucial to keep track of z-indices.

Reply:
👍 Extracted it into a separate variable and changed to 20, seems to work fine with this value.